pkg/apply/taskrunner/task.go

```diff
@@ -4,6 +4,7 @@
 package taskrunner
```

Comment:
Note: this file has so many changes that it's effectively a rewrite. It's probably easier to review the result, rather than the diff.